ci/cloudbuild/builds/cmake-single-feature.sh

```diff
@@ -52,6 +52,11 @@ check_pkgconfig_absolute() {
   fi
 }
 
+for feature in __ga_libraries__ __experimental_libraries__; do
```

Comment:
What does this do that the loop below does not?

Reply:
The loop below does too much.  It tries to verify `google_cloud_cpp_${feature}.pc` files have certain contents, that does not apply to `__ga_libraries__` and friends.

Reply:
D'oh. I somehow read this as looping over each library in the GA + experimental list. Not the strings "__ga_libraries__", "__experimental_libraries__". My bad.